zhaquirks/ikea/opencloseremote.py

```diff
@@ -0,0 +1,89 @@
+"""Device handler for IKEA of Sweden TRADFRI remote control."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.closures import WindowCovering
+from zigpy.zcl.clusters.general import (
+    Alarms,
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    PollControl,
+    PowerConfiguration,
+)
+from zigpy.zcl.clusters.lightlink import LightLink
+
+from . import IKEA
+from .. import DoublingPowerConfigurationCluster
+from ..const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+IKEA_CLUSTER_ID = 0xFC7C  # decimal = 64636
+
+
+class IkeaTradfriOpenCloseRemote(CustomDevice):
+    """Custom device representing IKEA of Sweden TRADFRI remote control."""
+
+    signature = {
+        MODELS_INFO: [("\x02KE", "TRADFRI open/close remote")],
```

Comment:
Wait, is it actually reported as "\x02KE"

Reply:
This is the info for my remote:
```
  "manufacturer": "\u0002KE",
  "model": "TRADFRI open/close remote"
```
Hope it helps!